src/RadioButton/RadioButton.tsx

```diff
@@ -0,0 +1,79 @@
+import React, {FC, ChangeEvent} from 'react';
+import styled from 'styled-components';
+
+import {Box} from '../Box';
+
+import {theme} from '../theme';
+
+interface IFakeInput {
+  checked?: boolean;
+}
+
+type RadioButtonProps = {
+  id: string;
+  label: string;
+  checked?: boolean;
+  value: string;
+  type?: string;
+  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+};
+
+export const RadioButton: FC<RadioButtonProps> = ({
+  id,
+  type = 'radio',
+  label,
+  checked,
+  value,
+  onChange,
+}) => (
+  <Box flex alignItems="center" mb="24px">
+    <RadioInput
+      id={id}
+      type={type}
+      checked={checked}
+      value={value}
+      onChange={onChange}
+    />
+
+    <RadioLabel htmlFor={id}>
+      <FakeInput checked={checked} />
+      <Text>{label}</Text>
+    </RadioLabel>
+  </Box>
+);
+
+const FakeInput = styled.div<IFakeInput>`
+  width: 24px;
+  height: 24px;
+  border-radius: 50%;
+  background-color: ${theme.colors.white};
+  flex-shrink: 0;
+  margin-right: 10px;
+  border: ${({checked}) =>
+    checked
+      ? `8px solid ${theme.colors.green5}`
+      : `1px solid ${theme.colors.grey4}`};
+`;
+
+const RadioInput = styled.input`
+  position: absolute;
+  opacity: 0;
+  height: 0;
+  width: 0;
+`;
+
+const RadioLabel = styled.label`
+  font-family: 'Gordita', sans-serif;
+  text-transform: none;
+  font-size: 16px;
+  line-height: 17px;
+  color: ${theme.colors.blue7};
+  margin-bottom: 0;
+  display: flex;
+`;
+
+// needed because the text is top aligned in its viewbox
+const Text = styled.span`
+  position: relative;
+  bottom: -5px;
+`;
```

Comment:
Potentially reduce the styles once using an imported `<Text />` (see comments above)

If custom positioning is required feel free to override via `styled(Text)`